sdk/servicebus/azure-servicebus/tests/async_tests/test_sb_client_async.py

```diff
@@ -58,3 +60,31 @@ async def test_async_sb_client_close_spawned_handlers(self, servicebus_namespace
         assert not sender._handler and not sender._running
         assert not receiver._handler and not receiver._running
         assert len(client._handlers) == 0
+
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
```

Comment:
do we need both of them?

Reply:
Yup.  Serve different purposes.  the latter is for EH/SB un-recordable livetests.